tests/models/owlvit/test_modeling_owlvit.py

```diff
@@ -91,6 +91,7 @@ def __init__(
 
     def prepare_config_and_inputs(self):
         pixel_values = floats_tensor([self.batch_size, self.num_channels, self.image_size, self.image_size])
+        pixel_values = pixel_values.to(torch_device)
```

Comment:
No need for this line, `floats_tensor` will automatically already put it on `torch_device` :)